compress/README.md

```diff
@@ -3,21 +3,22 @@
 ## About
 util-compress is a library for compressing and uncompressing data. Includes snappy and gzip.
 
-A binary is included for Linux-amd64. It will not work on 32-bit platforms. Other platforms will need to compile from source and may not compile or work as-is.
+A binary is included for Linux-amd64 and Linux-aarch64. It will not work on 32-bit platforms. Other platforms will need to compile from source and may not compile or work as-is.
```

Comment:
We still appear to have [macOS binaries](https://github.com/indeedeng/util/tree/main/compress/src/main/resources/native/Mac%20OS%20X), don't we?.

Reply:
Yes, I will add the bit about MacOS, it wasn't in the original(though is listed down below). I also noticed we don't mention ZSTD in the list of supported codecs.